docs/API.md

```diff
@@ -1606,89 +1606,67 @@ minioClient.getObjectTagging(
 
 <a name="getObjectLegalHold"></a>
 
-### getObjectLegalHold(bucketName, objectName, getOpts [, callback])
+### getObjectLegalHold(bucketName, objectName, getOpts)
 
 Get legal hold on an object.
 
 **Parameters**
 
-| Param           | Type       | Description                                                                                                            |
-| --------------- | ---------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `bucketName`    | _string_   | Name of the bucket.                                                                                                    |
-| `objectName`    | _string_   | Name of the object.                                                                                                    |
-| `getOpts`       | _object_   | Legal hold configuration options. e.g `{versionId:'my-version-uuid'}` defaults to `{}` .                               |
-| `callback(err)` | _function_ | Callback function is called with non `null` value in case of error. If no callback is passed, a `Promise` is returned. |
+| Param        | Type     | Description                                                                              |
+| ------------ | -------- | ---------------------------------------------------------------------------------------- |
+| `bucketName` | _string_ | Name of the bucket.                                                                      |
+| `objectName` | _string_ | Name of the object.                                                                      |
+| `getOpts`    | _object_ | Legal hold configuration options. e.g `{versionId:'my-version-uuid'}` defaults to `{}` . |
```

Comment:
Similar to my previous comment, can we also use a dot here to prevent confusion?

```suggestion
| `getOpts`    | _object_ | Legal hold configuration options. e.g `{versionId:'my-version-uuid'}`. Defaults to `{}` . |
```